pinot-segment-local/src/test/java/org/apache/pinot/segment/local/segment/index/forward/SortedForwardIndexReaderTest.java

```diff
@@ -28,8 +29,9 @@
 
 
 @Test
-public class SortedForwardIndexReaderTest {
+public class SortedForwardIndexReaderTest implements PinotBuffersAfterMethodCheckRule {
 
+  @Test
```

Comment:
Does this means that this tests wasn't executed before?

Reply:
No, I think I added it for consistency.